landing_page_backend/landing_page_backend/.env.example

```diff
@@ -0,0 +1,5 @@
+DEBUG=on
+SECRET_KEY=
+EMAIL_HOST_USER=
+EMAIL_RECIPIENT=
+EMAIL_HOST_PASSWORD=
```

Comment:
File should have single enter as a final symbol. No less, than 1, no more than 1.